tree/dataframe/src/RJittedAction.cxx

```diff
@@ -94,7 +94,7 @@ std::unique_ptr<ROOT::Detail::RDF::RMergeableValueBase> RJittedAction::GetMergea
    return fConcreteAction->GetMergeableValue();
 }
 
-std::function<void(unsigned int)> RJittedAction::GetDataBlockCallback()
+ROOT::RDF::DataBlockCallback_t RJittedAction::GetDataBlockCallback()
```

Comment:
Why is this (everywhere) not `ROOT::RDF::SampleCallback_t RJittedAction::GetSampleCallback()`, for naming consistency? (Mostly really a question, maybe a suggestion, but probably not something worth addressing in this PR.)

Reply:
yes I agree with the suggestion, see discussion at https://mattermost.web.cern.ch/root/pl/ztatm1f85fgx7f69bye7f51t5w

I'm considering switching to "sample" everywhere (`SampleCallback_t` and `RSampleInfo` instead of `RDataBlockId`)

Reply:
switched to "sample" everywhere